### PR TITLE
#161255186 setup redux 

### PR DESCRIPTION
**Pull request by p8ul**
#### What does this PR do?
- Setup react redux
#### Description of Task to be completed?
- Setup react redux

#### How should this be manually tested?
After cloning the repo, cd into it and RUN ``` npm test ```

#### What are the relevant pivotal tracker stories?
[#161255186](https://www.pivotaltracker.com/story/show/161255186)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,6 +4,8 @@
 /node_modules
 /.pnp
 .pnp.js
+package-lock.json
+yarn.lock
```
Comment (Rodgers-M) [inline .gitignore]: this should not be in `.gitignore`

Comment (p8ul) [reply .gitignore]: I have removed them from the file.

**Review by mmosoroohh — COMMENTED** [review]
So far looks good, work on @Rodgers-M has suggested. I think you will be good to continue...

**Comment by Tony-Ndichu** [comment]
I like the separation of concerns, that'll make our workflow easier

**Comment by Mirriam-Maina** [comment]
The changed files are comprehensive and the redux folders separate the concerns clearly

**Review by Rodgers-M — APPROVED** [review]

**Review by MartinKaburu — COMMENTED** [review]
Great job, I especially liked the depth of your tests.

**Pull request merged**